hack/run-functests.sh

```diff
@@ -17,6 +17,14 @@ export TESTS_REPORTS_PATH="${TESTS_REPORTS_PATH:-/tmp/artifacts/}"
 export failed=false
 export failures=()
 
+#env variables needed for the containerized version
+export TEST_POD_IMAGES_REGISTRY="${TEST_POD_IMAGES_REGISTRY:-quay.io/openshift-kni/}"
+export TEST_POD_CNF_TEST_IMAGE="${TEST_POD_CNF_TEST_IMAGE:-cnf-tests:4.5}"
+export TEST_POD_DPDK_TEST_IMAGE="${TEST_POD_DPDK_TEST_IMAGE:-dpdk:4.5}"
+
+export CNF_TESTS_IMAGE=$TEST_POD_IMAGES_REGISTRY$TEST_POD_CNF_TEST_IMAGE
```

Comment:
this is disabling any (easy) way of running the tests locally
I guess we'd better introduce yet another env like RUN_LOCALLY
and replace the if in L#32

Reply:
oh sorry, I see you already did that with $TESTS_IN_CONTAINER, ok

Reply:
I already introduced a new variable which is `TESTS_IN_CONTAINER`
If you look at line 40 below, it does what you are saying but with a different default